vendor/assets/javascripts/eui-1.0.0/eui.js

```diff
@@ -0,0 +1,199 @@
+/*jslint indent: 2 */
```

Comment:
is there a 1.0.9 release to js?

Reply:
So I ended up getting a hold of Jeff and the latest is in fact 1.1.6 so tried that and no issues, so going to keep it.   He said 1.1.7 is coming out later today, so might upgrade once again.